arm-datalake-analytics/catalog/2016-11-01/swagger/catalog.json

```diff
@@ -2767,11 +3343,12 @@
           "description": "the assembly file type.",
           "enum": [
             "Assembly",
-            "Resource"
+            "Resource",
+            "Nodeploy"
           ],
           "x-ms-enum": {
             "name": "FileType",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
See the description in the PR:

(BREAKING) Switch FileType enum to be modeled as a string. This is
important since this field changes frequently and we don't want to break
the SDK for this read-only property